tests/foreman/ui/test_activationkey.py

```diff
@@ -55,14 +55,14 @@ def setUpClass(cls):
         super(ActivationKey, cls).setUpClass()
 
     def create_sync_custom_repo(self, product_name=None, repo_name=None,
-                                repo_url=None, repo_type=None):
+                                repo_url=None, repo_type=None, org_id=None):
         """Create product/repo, sync it and returns repo_id"""
         product_name = product_name or gen_string("alpha", 8)
         repo_name = repo_name or gen_string("alpha", 8)
         # Creates new product and repository via API's
         product_attrs = entities.Product(
             name=product_name,
-            organization=self.org_id
+            organization=org_id or self.org_id
```

Comment:
It's Type is Integer.

```
(robottelo)[kedar@localhost robottelo]$ python
Python 2.7.3 (default, Aug  9 2012, 17:23:57) 
[GCC 4.7.1 20120720 (Red Hat 4.7.1-5)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> from robottelo import entities
/home/kedar/envs/robottelo/lib/python2.7/site-packages/fauxfactory/__init__.py:737: Warning: The FauxFactory class is deprecated. Please use functions from the fauxfactory module instead.
  category=Warning)
>>> org = entities.Organization().create()
2015-01-28 18:31:13 - py.warnings - WARNING - /home/kedar/envs/robottelo/lib/python2.7/site-packages/requests/packages/urllib3/connectionpool.py:734: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.org/en/latest/security.html
  InsecureRequestWarning)

>>> org['id']
259
>>> type(org['id'])
<type 'int'>
```

@elyezer,  As we are passing here `org_id` as data and not :id for path creation here, we need to pass it as string and hence the str conversion.